### DISCUSSION TITLE
Configuration minimum

### DISCUSSION THREAD

**Discussion by casa10** [madMAx43v3r/chia-plotter#155] [General]
Hello,
which configuration minimum is needed ? Because i have impression all mixed between DDR RAM and SSD or HDD.

For example : 
CPU : 12 cors/24 threads
RAM : 64 GB
SSD : 2Tb
OS supported : Centos, Ubuntu, etc.

How does work really ? Because that write only one time on hard disk (on the two temp directories) or that will use 1.8Tb of writting on your disk like swar, gui chia etc. ?

Thank  you

**Comment by madMAx43v3r** [comment]
It's fine, however 128GB RAM would be better

**Reply by casa10** [reply]
How i can use my 64GiB of ddr RAM and another SSD of 2TB for example ?
How i can configure all these things. I am beginner and i try to understand this way to do to use less SSD (keep TBW)

**Reply by madMAx43v3r** [reply]
To reduce SSD wear, you need at least 128GB RAM

**Reply by casa10** [reply]
Otherwise your program if you use only SSD because no 128 GiB of RAM, does he write 1,8Tb on the SSD like the other program of plotting (chia, swar, plotman) because you will use tmp1 and tmp2 on SSD?

**Reply by madMAx43v3r** [reply]
yes...

**Reply by casa10** [reply]
thank you for your answers
